src/Service/Image.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace App\Service;
+
+use App\Exception\FilesystemException;
+use App\Exception\ImageFetchException;
+use Liip\ImagineBundle\Message\WarmupCache;
+use Symfony\Component\Filesystem\Exception\IOExceptionInterface;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Filesystem\Path;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+final class Image implements ImageInterface
```

Comment:
```suggestion
final class ImageHandler implements ImageInterface
```

`Image` doesn't really tell anything about what this class does.